src/main/java/command/FindCommand.java

```diff
@@ -0,0 +1,43 @@
+package command;
+
+import cheatsheet.CheatSheet;
+import cheatsheet.CheatSheetList;
+import parser.ArgumentFlagEnum;
+import parser.Parser;
+
+public class FindCommand extends Command {
+    public FindCommand(Parser parser) {
+        super(parser);
+    }
+
+    @Override
+    public void execute() {
+        String programmingLanguage = "";
+        String keyword = "";
+
+        for (CheatSheet cs : CheatSheetList.getCheatSheetList()) {
+            if (parser.getDescriptionMap().containsKey(ArgumentFlagEnum.PROGRAMMINGLANGUAGE)) {
+                programmingLanguage = parser.getDescriptionMap().get(ArgumentFlagEnum.PROGRAMMINGLANGUAGE);
+            }
+            if (parser.getDescriptionMap().containsKey(ArgumentFlagEnum.SECTIONKEYWORD)) {
+                keyword = parser.getDescriptionMap().get(ArgumentFlagEnum.SECTIONKEYWORD);
+            }
```

Comment:
I think this one should be done by the constructor, or at least outside the for loop.